README.md

```diff
@@ -60,4 +58,3 @@ However, Tox does use [TCP relays](/docs/TCP_Network.txt) as a fallback if it en
 
 ### Connecting & Communicating
 Every peer is represented as a [byte string](https://en.wikipedia.org/wiki/String_(computer_science)) (the public key [Tox ID] of the peer). By using torrent-style DHT, peers can find the IP of other peers by using their Tox ID. Once the IP is obtained, peers can initiate a [secure](/docs/updates/Crypto.md) connection with each other. Once the connection is made, peers can exchange messages, send files, start video chats, etc. using encrypted communications.
-
```

Comment:
`EOF` is not a thing. This change is replacing `\n\n` at the end of the file with `\n`, which is a good change.